client/templates/website-template/src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Is this hardcoded English?

Reply:
oh god wikidot oh no I didn't mean it I just copied this from somewhere and forgot to change it